config/_default/menus/main.en.yaml

```diff
@@ -2873,6 +2873,36 @@ menu:
       parent: tracing_services
       identifier: tracing_apm_monitors
       weight: 706
+    - name: LLM Observability
+      url: tracing/llm_observability/
+      parent: tracing
+      identifier: tracing_llm_obs
+      weight: 7
+    - name: Quickstart
+      url: tracing/llm_observability/quickstart/
+      parent: tracing_llm_obs
+      identifier: tracing_llm_obs_quickstart
+      weight: 701
+    - name: Tracing LLM Applications
```

Comment:
is "Setup and tracing LLM Applications" too wordy? I think it's clearer to 1st time readers or readers with 0 context about LLM Observability altogether what this is about

Reply:
I may be cynical on this one, but I personally think simpler is better. We could use "Trace an LLM application" if that makes it clearer that this is an action they can take, and we will guide them through it (including setup, of course)?

Reply:
Yeah let's start with "Trace an LLM application" then